templates/zerver/help/mute-a-topic.md

```diff
@@ -72,3 +72,18 @@ unmute the topic that the selected message belongs to.
 the topic will reappear in your filters and the topic will stop being grayed
 out in the **Streams** section, confirming the success of your unmuting of the
 topic.
+
+## Unmute a topic through the Settings page
+
+{!go-to-the.md!} [Muted topics](/#settings/muted-topics)
+{!settings.md!}
+
+2. Locate the topic you want to unmute in the **Muted topics** section; the
+topics are sorted according to their corresponding streams by age, oldest to
```

Comment:
Yeah, looks like they're sorted by when you first mute the stream, thanks for the catch.